### DISCUSSION TITLE
Import from installed apps on phone

### DISCUSSION THREAD

**Discussion by fakhamatia** [ImranR98/Obtainium#864] [Ideas]
Import from installed apps on phone

**Comment by hirenshah** [comment]
This would definitely be very useful.

**Comment by ImranR98** [comment]
This was already suggested in #163 (please search the issues before opening a separate thread). The problem is that we need a way to get app urls from package ids. I have reopened the issue as it might be doable using a combination of fdroid (as someone proposed in a comment) and github search.
